File: Example/IQTextView_ObjcExample/ViewController.m
//
//  ViewController.m
//  IQTextView_ObjcExample
//
//  Created by Iftekhar on 8/8/24.
//  Copyright © 2024 CocoaPods. All rights reserved.
//

#import "ViewController.h"
#import <IQTextView/IQTextView-Swift.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    IQTextView *textView = [[IQTextView alloc] init];
    [textView placeholderLabel];
    [textView placeholderTextColor];
    [textView placeholder];
    [textView attributedPlaceholder];
    [textView text];
    [textView attributedText];
}


@end

File: Example/IQTextView/ViewController.swift
//
//  ViewController.swift
//  IQTextView
//
//  Created by hackiftekhar on 07/22/2024.
//  Copyright (c) 2024 hackiftekhar. All rights reserved.
//

import UIKit
import IQTextView

class ViewController: UIViewController {

    @IBOutlet private var textView: IQTextView!

    override func viewDidLoad() {
        super.viewDidLoad()

        textView.contentInset = .init(top: 8, left: 8, bottom: 8, right: 8)
        // Do any additional setup after loading the view, typically from a nib.
    }
}
